type_go_map_test.go

```diff
@@ -0,0 +1,37 @@
+package otto
+
+import (
+	"testing"
+	"fmt"
+	"sort"
+)
+
+type GoMapTest map[string]int
+
+func (s GoMapTest) Join() string {
+	joinedStr := ""
+
+	//Ordering the map takes some effort
```

Comment:
Space after //
Also update the comment so it describes why your doing the sort.